classes/admin/saml2_settings.php

```diff
@@ -41,4 +41,12 @@ abstract class saml2_settings {
     const OPTION_MULTI_IDP_DISPLAY_DROPDOWN = 0;
 
     const OPTION_MULTI_IDP_DISPLAY_BUTTONS = 1;
+
+    const OPTION_FLAGGED_LOGIN_NONE = 0;
+
+    const OPTION_FLAGGED_LOGIN_MESSAGE = 1;
+
+    const OPTION_FLAGGED_LOGIN_REDIRECT = 2;
+
+    const FLAG_ATTRIBUTE_MAX_LENGTH = 50;
```

Comment:
This was an arbitrary limitation to prevent overly long strings, Do you think I should change to 255, standard string size without the line return? Or remove the limitation entirely?

Reply:
I'm not sure about this one. What's wrong with long strings? Is there any restrictions in SAML about that?

Reply:
I will remove the restriction on the field length and this constant